recipes/neptune-notebooks/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "neptune-notebooks" %}
+{% set version = "0.0.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/neptune-ai/neptune-notebooks/archive/{{ version }}.tar.gz"
+  sha256: c225822336fddb524f81130eb877727465b9947ab69a6bf84b75c5a79ee8abf0
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - neptune_notebooks = neptune_notebooks_plugin:notebook
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - click >=7.0
+    - future >=0.17.1
+    - neptune-client >=0.4.92
+    - notebook >=4.3.0
+    - path.py
+
+test:
+  imports:
+    - neptune_notebooks
+    - tests.neptune_notebooks
+    - tests.neptune_notebooks_plugin
+
+about:
+  home: "https://neptune.ai/"
+  license: Apache-2.0
+  license_family: "APACHE"
+  license_file: "LICENSE"
+  summary: "Neptune's integration with Jupiter and JupiterLab notebooks"
+  doc_url: "https://docs.neptune.ai"
+  dev_url: "https://github.com/neptune-ai/neptune-notebooks"
+
+extra:
+  recipe-maintainers:
+    - szymon-kuklewicz
+    - aniezurawski
+    - hubertjaworskineptune
+    - pitercl
```

Comment:
These other maintainers need to comment here saying they are ok doing maintenance. 

@aniezurawski @hubertjaworskineptune @pitercl